### PR TITLE
Add craigbox as Istio GitHub member

### PR DESCRIPTION
**Pull request by craigbox**
- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [x] I have joined [Istio's Slack workspace](https://istio.slack.com) 

List your company name, or indicate Individual if you're not affiliated with a company:
Google

Provide a link to at least one PR that you have successfully pushed to one
of the Istio repos:
https://github.com/istio/istio.io/pull/5206 (They'll get more interesting, I promise.)

### CODE REVIEW

File: org/istio.yaml [hunk 1]
```diff
@@ -44,6 +44,7 @@ orgs:
     - clyang82
     - cmluciano
     - costinm
+    - craigbox
```
Comment (sdake) [inline org/istio.yaml]: hi @craigbox ! Welcome to the Istio community. This list is ASCII-betical. Can you correct the ordering?

File: org/istio.yaml [hunk 2]
```diff
@@ -262,6 +263,7 @@ orgs:
         - clyang82
         - cmluciano
         - costinm
+        - craigbox
```
Comment (sdake) [inline org/istio.yaml]: same comment applies.

**Comment by howardjohn** [comment]
/ok-to-test

we have a test for alpha-ordering

**Comment by sdake** [comment]
> /ok-to-test
> 
> we have a test for alpha-ordering

thanks, apparently I forgot my ABCs.. 😀

**Comment by smawson** [comment]
Who is this "self" person and why are we allowing them into our carefully curated community?!? (Please change the PR description).

**Comment by craigbox** [comment]
Thanks all!

**Pull request merged**